src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return color + " " + number;
```

Comment:
```suggestion
        return "Ball with color: "+ color + " and number: " + number;
```

Reply:
previous was supper non informative, "red 3" doesn't seem like ball

Reply:
Changed this return statement